common-types/index.d.ts

```diff
@@ -42,24 +42,9 @@ interface IdolMemberDiff {
 }
 
 /** The data type used by Nova site to represent a DTI member. */
-interface NovaMember {
-  readonly netid: string;
-  readonly name: string;
-  readonly pronouns: string;
-  readonly isLead?: boolean;
-  readonly graduation: string;
-  readonly major: string;
-  readonly doubleMajor?: string;
-  readonly minor?: string;
-  readonly website?: string;
-  readonly linkedin?: string;
-  readonly github?: string;
-  readonly hometown: string;
-  readonly about: string;
-  readonly subteams?: string[];
-  readonly formerSubteams?: string[];
-  readonly roleId: string;
-  readonly roleDescription: string;
+interface NovaMember extends IdolMember {
```

Comment:
nit: Maybe change `NovaMember` to something more descriptive like `RenderableMember` or `MemberProfile`?